{"level":"error","message":"❌ Unable to connect to the database:","name":"SequelizeConnectionRefusedError","original":{"code":"ECONNREFUSED"},"parent":{"code":"ECONNREFUSED"},"stack":"SequelizeConnectionRefusedError\n    at Client._connectionCallback (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/client.js:336:19)\n    at Client._handleErrorEvent (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/client.js:346:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/connection.js:57:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-16T02:06:49.262Z"}
{"level":"error","message":"❌ Unable to connect to the database:","name":"SequelizeConnectionRefusedError","original":{"code":"ECONNREFUSED"},"parent":{"code":"ECONNREFUSED"},"stack":"SequelizeConnectionRefusedError\n    at Client._connectionCallback (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/client.js:336:19)\n    at Client._handleErrorEvent (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/client.js:346:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/connection.js:57:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-16T02:06:49.294Z"}
{"level":"error","message":"❌ Unable to connect to the database:","name":"SequelizeConnectionRefusedError","original":{"code":"ECONNREFUSED"},"parent":{"code":"ECONNREFUSED"},"stack":"SequelizeConnectionRefusedError\n    at Client._connectionCallback (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/client.js:336:19)\n    at Client._handleErrorEvent (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/client.js:346:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/connection.js:57:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-16T02:07:39.891Z"}
{"level":"error","message":"❌ Unable to connect to the database:","name":"SequelizeConnectionRefusedError","original":{"code":"ECONNREFUSED"},"parent":{"code":"ECONNREFUSED"},"stack":"SequelizeConnectionRefusedError\n    at Client._connectionCallback (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/client.js:336:19)\n    at Client._handleErrorEvent (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/client.js:346:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/connection.js:57:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-16T02:07:39.900Z"}
{"level":"error","message":"❌ Unable to connect to the database:","name":"SequelizeConnectionRefusedError","original":{"code":"ECONNREFUSED"},"parent":{"code":"ECONNREFUSED"},"stack":"SequelizeConnectionRefusedError\n    at Client._connectionCallback (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/client.js:336:19)\n    at Client._handleErrorEvent (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/client.js:346:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/workspaces/ProjectMafiest-final/MaFiEst/mafiest_backend/node_modules/pg/lib/connection.js:57:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-16T02:07:45.925Z"}
